### PR TITLE
net-dns/knot: Version bump

### PR DESCRIPTION
**Pull request by nemunaire**
No news from the maintainer since my report in bug [#581568](https://bugs.gentoo.org/show_bug.cgi?id=581568); so here it is my pull-request for the version bump to latest upstream releases: both stable (1.6.8) and current (2.3.0).

The 2.3.0 release fixes CVE-2016-6171: related to [#588654](https://bugs.gentoo.org/show_bug.cgi?id=588654).

### CODE REVIEW

File: net-dns/knot/knot-1.6.8.ebuild [hunk 1]
```diff
@@ -29,13 +32,14 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
 	sys-devel/flex
 	sys-devel/bison
-	doc? ( dev-python/sphinx app-text/texlive-core sys-apps/texinfo )
+	doc? ( dev-python/sphinx )
 "
 
 S="${WORKDIR}/${P/_/-}"
 
 src_prepare() {
```
Comment (SoapGentoo) [inline net-dns/knot/knot-1.6.8.ebuild]: put the patch in the `PATCHES` variable and then obliterate `src_prepare()`

File: net-dns/knot/knot-1.6.8.ebuild [hunk 2]
```diff
@@ -53,40 +57,28 @@ src_configure() {
...
 
 	if use doc; then
-		dodoc doc/_build/latex/KnotDNS.pdf
-
 		docinto html
 		dodoc doc/_build/html/*.html doc/_build/html/*.js
 		docinto html/_sources
 		dodoc doc/_build/html/_sources/*
 		docinto html/_static
 		dodoc doc/_build/html/_static/*
```
Comment (SoapGentoo) [inline net-dns/knot/knot-1.6.8.ebuild]: replace the whole if use doc conditional with:
```
use doc && HTML_DOCS=( doc/_build/html/{*.html,*.js,_sources,_static} )
einstalldocs
```

File: net-dns/knot/knot-2.3.0.ebuild [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```
Comment (SoapGentoo) [inline net-dns/knot/knot-2.3.0.ebuild]: do the same edits for this bump too

File: net-dns/knot/knot-1.6.8.ebuild (hunk 2 above)
Comment (SoapGentoo) [inline net-dns/knot/knot-1.6.8.ebuild]: and put it before the default, then you don't even need `einstalldocs`, as that is implicitly called by `default_src_install` in `EAPI=6`

**Comment by nemunaire** [comment]
Thanks for your review @SoapGentoo! I've updated my pull request, following your advises.

**Comment by gktrk** [comment]
@scarabeusiv @gentoo/proxy-maint

**Pull request closed without merging**